pkg/job/abstract.go

```diff
@@ -199,13 +198,8 @@ func scrapeStaticJob(ctx context.Context, resource *config.Static, region string
 	return cw
 }
 
-func getMetricDataInputLength(job *config.Job) int64 {
-	length := model.DefaultLengthSeconds
-
-	if job.Length > 0 {
-		length = job.Length
-	}
-	for _, metric := range job.Metrics {
+func getMetricDataInputLength(metrics []*config.Metric) (length int64) {
```

Comment:
Could you please use an explicit `length :=0` instead of the named return param? 🙏